plasmapy/analysis/fit_functions.py

```diff
@@ -633,7 +633,7 @@ def curve_fit(self, xdata, ydata, **kwargs) -> None:
             The dependent data associated with ``xdata``.
 
         **kwargs
-            Any keywords accepted by `scipy.stats.linregress.curve_fit`.
+            Any keywords accepted by `scipy.optimize.curve_fit`.
```

Comment:
Since `**kwargs` isn't used in the function, I'm not sure if this should be for `scipy.stats.linregress` or `scipy.optimize.curve_fit`.  Is this correct?